crypto/rand/randfile.c

```diff
@@ -104,6 +104,12 @@ static __FILE_ptr32 (*const vms_fopen)(const char *, const char *, ...) =
 
 #define RFILE ".rnd"
 
+#ifdef EINTR
+# define INTERRUPTED(in) (ferror(in) && errno == EINTR)
```

Comment:
Specification (and manual) says nothing about `errno` being reliable way to identify error condition in case of ferror... Hmmm....